tests/System.Buffers.Experimental.Tests/BytesReaderTests.cs

```diff
@@ -16,14 +16,14 @@ public void SingleSegmentBytesReader()
             ReadOnlyBytes bytes = Create("AB CD#EF&&");
             var reader = BytesReader.Create(bytes);
 
-            var ab = bytes.Slice(reader.ReadRange((byte)' '));
+            Assert.True(reader.TryReadBytes(out var ab, (byte)' '));
             Assert.Equal("AB", ab.ToString(SymbolTable.InvariantUtf8));
 
-            var cd = bytes.Slice(reader.ReadRange((byte)'#'));
+            Assert.True(reader.TryReadBytes(out var cd, (byte)'#'));
             Assert.Equal("CD", cd.ToString(SymbolTable.InvariantUtf8));
 
-            var ef = bytes.Slice(reader.ReadRange(new byte[] { (byte)'&', (byte)'&' }));
-            Assert.Equal("EF", ef.ToString(SymbolTable.InvariantUtf8));
+            //Assert.True(reader.TryReadBytes(out var ef, new byte[] { (byte)'&', (byte)'&' }));
```

Comment:
Why is this commented out rather than removed (same for other places)?

Reply:
Will fix.